get-data/main.go

```diff
@@ -0,0 +1,328 @@
+	"time"
+
+	"cloud.google.com/go/firestore"
+	firebase "firebase.google.com/go"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ssm"
+	"google.golang.org/api/iterator"
+	"google.golang.org/api/option"
+)
+
+/*
+ Structures
+*/
+type Log struct {
+	Type      string    `firestore:"type,omitempty"`
+	Timestamp time.Time `firestore:"timestamp,omitempty"`
+	Body      string    `firestore:"body,omitempty"`
+}
+
+type Res struct {
+	FirstName   string `json:"first_name"`
+	LastName    string `json:"last_name"`
+	Email       string `json:"email"`
+	Phone       string `json:"phone"`
+	YOE         int    `json:"yoe"`
+	Company     string `json:"company"`
+	Designation string `json:"designation"`
+	GithubId    string `json:"github_id"`
+	LinkedIn    string `json:"linkedin_id"`
+	TwitterId   string `json:"twitter_id"`
+	InstagramId string `json:"instagram_id"`
+	Website     string `json:"website"`
+}
+
+type Diff struct {
+	Username    string    `firestore:"username,omitempty"`
+	Timestamp   time.Time `firestore:"timestamp,omitempty"`
+	Approval    string    `firestore:"approval"`
+	FirstName   string    `firestore:"first_name,omitempty"`
+	LastName    string    `firestore:"last_name,omitempty"`
+	Email       string    `firestore:"email,omitempty"`
+	Phone       string    `firestore:"phone,omitempty"`
+	YOE         int       `firestore:"yoe,omitempty"`
+	Company     string    `firestore:"company,omitempty"`
+	Designation string    `firestore:"designation,omitempty"`
+	GithubId    string    `firestore:"github_id,omitempty"`
+	LinkedIn    string    `firestore:"linkedin_id,omitempty"`
+	TwitterId   string    `firestore:"twitter_id,omitempty"`
+	InstagramId string    `firestore:"instagram_id,omitempty"`
+	Website     string    `firestore:"website,omitempty"`
+}
+
+/*
+ Structures Conversions
+*/
+func diffToRes(diff Diff) Res {
+	return Res{
+		FirstName:   diff.FirstName,
+		LastName:    diff.LastName,
+		Email:       diff.Email,
+		Phone:       diff.Phone,
+		YOE:         diff.YOE,
+		Company:     diff.Company,
+		Designation: diff.Designation,
+		GithubId:    diff.GithubId,
+		LinkedIn:    diff.LinkedIn,
+		TwitterId:   diff.TwitterId,
+		InstagramId: diff.InstagramId,
+		Website:     diff.Website,
+	}
+}
+
+func resToDiff(res Res, username string) Diff {
+	return Diff{
+		Username:    username,
+		Timestamp:   time.Now(),
+		Approval:    "PENDING",
+		FirstName:   res.FirstName,
+		LastName:    res.LastName,
+		Email:       res.Email,
+		Phone:       res.Phone,
+		YOE:         res.YOE,
+		Company:     res.Company,
+		Designation: res.Designation,
+		GithubId:    res.GithubId,
+		LinkedIn:    res.LinkedIn,
+		TwitterId:   res.TwitterId,
+		InstagramId: res.InstagramId,
+		Website:     res.Website,
+	}
+}
+
+/*
+ Setting Firestore Key for development/production
+*/
+func getFirestoreKey() string {
+	if os.Getenv(("environment")) == "DEVELOPMENT" {
```

Comment:
Could we be creating new constants?go file to import the package and utilize it here?